docs/app/pages/components/components-sections/wizard/marquee-wizard/marquee-wizard.component.html

```diff
@@ -258,9 +258,9 @@ <h1 class="m-t-nil">Sixth step title</h1>
     <tr uxd-api-property name="finishAriaLabel" type="string">
         Defines the text for the aria label on the 'Finish' button.
     </tr>
-    <tr uxd-api-property name="nextDisabled" type="boolean">
+    <tr uxd-api-property name="disableNextWhenInvalid" type="boolean">
```

Comment:
Did you forget to define this on the `WizardComponent`? The recommendation was to use the `disableNextWhenInvalid` value from `WizardComponent`, optionally overridden by the value on `WizardStepComponent`.